hack/install-etcd.sh

```diff
@@ -30,11 +30,17 @@ else
   pushd etcd >/dev/null
 fi
 
+# setup a private GOPATH so the build can succeed
+export GOPATH=${PWD}/gopath
+rm -f ${GOPATH}/src/github.com/coreos/etcd
```

Comment:
@ncdc still would prefer to quote expansions - never know when someone's `pwd` has a space/newline/whatever in it